main.css

```diff
@@ -0,0 +1,500 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+}
+
+.hamb-icon {
+  width: 30px;
+  height: 30px;
+  margin-top: 12px;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+.menu-links {
+  height: 85%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 0 0 10px 10px;
+  background-color: #fff;
+  padding-top: 40px;
+  display: none;
+  flex-direction: column;
+  box-shadow: #ec5242 10px 10px 90px 40px;
+  margin: 0;
+}
+
+.menu-list {
+  margin: 20px 0;
+}
+
+.hamb-close-btn {
+  width: 35px;
+  height: 35px;
+  padding-right: 20px;
+}
+
+.menu-list.close {
+  align-self: flex-end;
+  margin-top: 0;
+}
+
+.menu-link {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  text-transform: uppercase;
+  font-size: 26px;
+  color: #ec5242;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+header,
+.headline {
+  background-image: url(background1.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.headline.about-me-page {
+  align-items: center;
+}
+
+.main-title::before {
+  content: '"Hello! Sharing World"';
+  text-transform: none;
+  display: block;
+  font-size: 20px;
+  font-weight: normal;
+  font-family: 'lato', sans-serif;
+}
+
+.main-title.about-me-page::before {
+  text-align: center;
+}
+
+.main-title {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  text-transform: uppercase;
+  font-size: 35px;
+  color: #ec5242;
+  margin-left: 15px;
+}
+
+.main-title.about-me-page {
+  text-align: center;
+}
+
+.headline-description {
+  font-family: 'lato', sans-serif;
+  font-size: 19px;
+  padding: 10px;
+  border: 2px solid #fff;
+  align-self: center;
+  margin: 0 15px;
+  color: #272a31;
+}
+
+.headline-description.about-me-page {
+  text-align: center;
+  background-color: #fff;
+  color: #272a31;
+}
```

Comment:
- [ ] To match the original design better, you should consider adding a border to this section in the desktop version.